timelock-server-distribution/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'com.palantir.java-distribution'
+
+apply from: "../gradle/docker.gradle"
+apply from: "../gradle/publish-dists.gradle"
+apply from: "../gradle/shared.gradle"
+
+dependencies {
+    runtime project(':timelock-server')
+    runtime group: 'org.mortbay.jetty.alpn', name: 'jetty-alpn-agent', version: libVersions.jetty_alpn_agent
+}
+
+distribution {
+    serviceName "timelock-server"
+    mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
+    args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-Xms512m", "-Xmx512m", "-javaagent:service/lib/jetty-alpn-agent-${libVersions.jetty_alpn_agent}.jar"
+}
+
+configurations.all {
+    resolutionStrategy {
+        eachDependency { details ->
+            //specifying a fixed version for all libraries with 'io.dropwizard' group
+            switch (details.requested.group) {
+                case 'io.dropwizard':
+                    details.useVersion libVersions.timelock_dropwizard
+                    break
+                case 'com.fasterxml.jackson.core':
+                case 'com.fasterxml.jackson.datatype':
+                    details.useVersion libVersions.timelock_jackson
+                    break
+                case 'org.eclipse.jetty':
+                    details.useVersion libVersions.timelock_jetty
+                    break
+                case 'org.glassfish.jersey.core':
+                    details.useVersion libVersions.timelock_jersey
+                    break
+                case 'com.google.guava':
+                    details.useVersion libVersions.timelock_guava
+                    break
+                case 'org.glassfish.hk2':
+                case 'org.glassfish.hk2.external':
+                    details.useVersion libVersions.glassfish_hk2
+                    break
```

Comment:
found a better fix.. actually I had missed copying
```
dependencyRecommendations {
    strategy ConflictResolved
}
```
Consolidated the two files to avoid such future mistakes :).